python/BUILD.bazel

```diff
@@ -167,6 +168,24 @@ internal_copy_files(
     strip_prefix = "src",
 )
 
+internal_copy_files(
```

Comment:
I can look into removing them entirely.  This was my first migration so I just went with the methods that were already in place.  I've learned a lot since then and think they might be avoidable altogether

Reply:
Unfortunately I still can't figure out a better solution here.  The proto2 and proto3 conformance test protos need to be kept separate, and once they're combined there's not a great way to split them back up.

I think this weirdness stems from our py_proto_library rule, but I'm hesitant to make any functional changes to it since it appears to be part of our public API (http://shortn/_85Vzaz3or8)